corehq/apps/users/migrations/0043_add_release_management_permission.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+from corehq.apps.users.models_role import SQLPermission
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
```

Comment:
Why skip on fresh install? I know permissions get created dynamically as well but I don't think there's another migration that will create the permission.